Classes/API/Drafts/Charts/WalletWealthBalanceChartDataDraft.swift

```diff
@@ -16,21 +16,15 @@
 
 import MagpieCore
 
-struct WalletWealthBalanceChartDataDraft: ObjectQuery {
+struct WalletWealthBalanceChartDataDraft: JSONObjectBody {
     var accountAddresses: [String]
     var period: ChartDataPeriod
-    var ordering: String?
     
-    var queryParams: [APIQueryParam] {
-        var params: [APIQueryParam] = []
-
-        params.append(APIQueryParam(.accountAddresses, accountAddresses))
-        
-        params.append(APIQueryParam(.period, period.rawValue))
-        
-        if let ordering {
-            params.append(APIQueryParam(.ordering, ordering))
-        }
+    var bodyParams: [APIBodyParam] {
+        var params: [APIBodyParam] = []
+
+        params.append(APIBodyParam(.accountAddresses, accountAddresses))
+        params.append(APIBodyParam(.period, period.rawValue))
 
         return params
```

Comment:
There is no need to app parameters one-by-one

```suggestion
    var bodyParams: [APIBodyParam] { [APIBodyParam(.accountAddresses, accountAddresses), APIBodyParam(.period, period.rawValue)] }
```